content/en/docs/tasks/traffic-management/ingress/gateway-api/index.md

```diff
@@ -3,7 +3,7 @@ title: Kubernetes Gateway API
 description: Describes how to configure the Kubernetes Gateway API with Istio.
 weight: 50
 aliases:
-    - /docs/tasks/traffic-management/ingress/service-apis/
+    - /latest/docs/tasks/traffic-management/ingress/service-apis/
```

Comment:
This doesn't look right. Won't this result in preliminary.istio.io redirecting to istio.io?

Also, this looks like a much more general problem ... aren't ALL aliases on the site broken this same way?

Reply:
cc: @davidhauck @craigbox

Reply:
While this is a general issue, I haven't noticed any other link broke other than this one because we recently renamed it.

Reply:
Agree that this is more of a recent problem as someone needed to save a link with `latest` and then have the page renamed. Earlier (pre using `latest`) URLs which were saved without `latest` in the link continue to work as the redirect from the non-`latest` links do continue to the new page.

Reply:
That said, people do need to realize that saving links with `latest` in them may not always point to the same info as a page may get updated release to release. While it may have pointed to the release-1.8 docs a few months ago, it might  point the the release-1.10 docs in another month.